cmd/migration-assist/commands/postgres.go

```diff
@@ -108,9 +111,35 @@ func runTargetCheckCmdF(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
+	mysqlMigrations := "mysql.output"
```

Comment:
It's also possible that the user might have renamed the output file right? Would it be better to pass the filename from CLI?